### PR TITLE
fix: broken quarkus dependencies

### PR DESCRIPTION
**Pull request by mwdchang**
# Description
Fix bad dependency where `quarkus dev` wouldn't start properly.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by Nithos** [comment]
@dgauldie @Tom-Szendrey can this be reviewed and merged please. Think this is blocking and causing some issues.

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #Gradle properties
 quarkusPluginId=io.quarkus
-quarkusPluginVersion=2.12.1.Final
+quarkusPluginVersion=2.13.2.Final
```
Comment (bigglesandginger) [inline gradle.properties:3]: should this not be the same version as `quarkusPlatformVersion` - ie `2.13.3.FInal`?

Comment (Nithos) [reply gradle.properties]: updated to match the bom

**Comment by Nithos** [comment]
So this I believe stems from the fact that Renovate Bot updated the BOM but not the plugin, should these always be in sync? If so we need to tell renovate to group these for future reference.

**Review by dgauldie — APPROVED** [review]

**Pull request merged**